.github/workflows/allthekeeps-testnet.yml

```diff
@@ -0,0 +1,103 @@
+name: AllTheKeeps / Testnet
+
+on:
+  push:
+    branches:
+     - allthekeeps-on-ropsten
+  pull_request:
+  workflow_dispatch:
+    inputs:
+      environment:
+        description: 'Environment for workflow execution'
+        required: false
+        default: 'dev'
+      upstream_builds:
+        description: 'Upstream builds'
+        required: false
+      upstream_ref:
+        description: 'Git reference to checkout (e.g. branch name)'
+        required: false
+        default: 'main'
+
+jobs:
+  dapp-build-and-publish:
+    if: github.event_name != 'pull_request'
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+        with:
+          ref: allthekeeps-on-ropsten
+
+      - name: Load environment variables
+        uses: keep-network/ci/actions/load-env-variables@v1
+        with:
+          environment: 'ropsten'
```

Comment:
We could try to use `environment` workflow_dispatch input variable.

Reply:
sorry, not sure what you mean. FWIW we have the same pattern across all other dApps in keep org.

Reply:
Kuba means using `environment: `${{ github.event.inputs.environment }}`.
We don't use that for other dApps, but we do in other workflows, for example [here](https://github.com/keep-network/keep-core/blob/c92d1d627323a3bf98c2aa8397bd9ca696bfb453/.github/workflows/contracts.yml#L128).
It wasn't used in dApps because prior to uploading dApp files to GCS, we were using the `Load environment variables` action not only on `workflow_dispatch` but also on other events (so we couldn't read `environment` from the `workflow_dispatch` input, as it would be empty for workflows run due to push or PR). 
Right now, when switched to GCS, we could use the `environment: `${{ github.event.inputs.environment }}` for all our dApps.

Reply:
By other dApps I meant that we have similar hardcoded network name in [keep-dashboard](https://github.com/keep-network/keep-core/blob/c92d1d627323a3bf98c2aa8397bd9ca696bfb453/.github/workflows/dashboard-testnet.yml#L59) and [tbtc-dapp](https://github.com/keep-network/keep-core/blob/c92d1d627323a3bf98c2aa8397bd9ca696bfb453/.github/workflows/dashboard-testnet.yml#L59). I'll change it to `environment: '${{ github.event.inputs.environment }}'`

Reply:
763c068